hazelcast/src/main/java/com/hazelcast/jet/impl/JobCoordinationService.java

```diff
@@ -497,6 +499,12 @@ public CompletableFuture<Void> terminateJob(long jobId, TerminationMode terminat
                                 + ", should be " + RUNNING);
                     }
 
+                    if ((terminationMode == SUSPEND_GRACEFUL || terminationMode == SUSPEND_FORCEFUL
+                            || terminationMode == RESTART_GRACEFUL || terminationMode == RESTART_FORCEFUL)
+                            && masterContext.jobConfig().getArgument(IS_JOB_SUSPENDABLE) != null) {
```

Comment:
if this argument looks like a boolean we should handle it as a boolean, someone might send IS_JOB_SUSPENDABLE=false